elisp-lint.el

```diff
@@ -362,6 +362,18 @@ Use a file variable or \".dir-locals.el\" to override the default value."
   "^;;[ \t]+Package-Requires:"
   "This regexp must match the definition in package.el.")
 
+(defvar elisp-lint--url-in-document-regexp
+  (concat "^"
+          "[[:blank:]]*"
+          "\\(?:;+\\|\"\\)?"
+          "[[:blank:]]*"
+          "https?://"
```

Comment:
Can we make this `[a-zA-Z][-a-zA-Z0-9+.]*:(//)?` instead, so that other protocols also match?

Reply:
hey, so I'm going to merge this PR first, and then one of us can make this change to recognize all valid URL schemes after that.

Reply:
You might also want to look at `goto-address-url-regexp` and the variables in thingatpt (e.g. `thing-at-point-uri-schemes`).

Reply:
`thingatpt` just hard-codes a list, unfortunately, which is explicitly not compliant with the RFC, but might work well enough in fairness.

I hadn't seen `goto-address-url-regexp` though :)